src/platform/apollolake/include/platform/memory.h

```diff
@@ -378,7 +378,7 @@
 #define HEAP_LP_BUFFER_BASE LP_SRAM_BASE
 #define HEAP_LP_BUFFER_SIZE LP_SRAM_SIZE
 
-#define HEAP_LP_BUFFER_BLOCK_SIZE	0x180
+#define HEAP_LP_BUFFER_BLOCK_SIZE	0x100
```

Comment:
I think this is good but invasive change, hope we won't have any side effect with this @lgirdwood ?

Reply:
Yep, this is part of @RanderWang global memory allocator PR's that still going through CI. This patch should be removed from this PR.